### PR TITLE
Add io.github.efogdev.mpris-timer

### PR DESCRIPTION
**Pull request by EfogDev**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly. 
A timer app (cli/GUI) utilizing GTK4, Libadwaita and MPRIS interface for seamless Gnome integration.
- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. https://github.com/EfogDev/mpris-timer

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by EfogDev** [comment]
bot, build io.github.efogdev.mpris-timer

File: io.github.efogdev.mpris-timer.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+sdk: org.gnome.Sdk
+command: mpris-timer
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=session-bus # for multiple instances
```
Comment (hfiguiere) [inline io.github.efogdev.mpris-timer.yml:13]: nope. This is not allowed. You have to determine the needed names instead.

File: misc/io.github.efogdev.mpris-timer.desktop
Comment (hfiguiere) [inline misc/io.github.efogdev.mpris-timer.desktop:1]: this belong upstream

File: misc/io.github.efogdev.mpris-timer.desktop [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Exec=/app/bin/mpris-timer
```
Comment (hfiguiere) [inline misc/io.github.efogdev.mpris-timer.desktop:3]: this is NEVER a path

File: misc/io.github.efogdev.mpris-timer.svg
Comment (hfiguiere) [inline misc/io.github.efogdev.mpris-timer.svg:1]: this belong upstream

**Review by hfiguiere — CHANGES_REQUESTED** [review]
Missing: the required appstream file.

**Comment by hfiguiere** [comment]
> Missing: the required appstream file.

which also should be upstream

**Pull request closed without merging**